pkgs/development/libraries/tracker/default.nix

```diff
@@ -73,10 +73,10 @@ stdenv.mkDerivation rec {
     libuuid
     json-glib
     libstemmer
+    python3.pkgs.pygobject3
   ];
 
   checkInputs = with python3.pkgs; [
-    pygobject3
     tappy
```

Comment:
Looks like the check was added in https://gitlab.gnome.org/GNOME/tracker/-/commit/8569c99d2926d55419affa6c5269053d5c4dc97a. And also it appears tappy is not needed any more https://gitlab.gnome.org/GNOME/tracker/-/merge_requests/385.